features/admins/users/admin_fnok.feature

```diff
@@ -15,6 +15,7 @@ Feature: Admin Fannish Next Of Kind actions
       And I fill in "Fannish next of kin's email" with "testy@foo.com"
       And I press "Update Fannish Next of Kin"
     Then I should see "Fannish next of kin was updated."
+    Then I should see "Fannish Next of Kin Added"
```

Comment:
This is going to be slightly annoying to do, but I think we need to be testing the entire string here to make sure we're including the right user ID on the right page. Please forgive my pseudo code, but maybe something like:

```
# Usage on harrykim's user admin page: 
# - Then adding libby as next of kin should be logged in the history table
# - Then removing libby as next of kin should be logged in the history table
Then "{string} {string} as next of kin should be logged in the history table" do |action, fnok_username|
  fnok_id = User.find_by(login: fnok).id
  step %{I should see "Fannish Next of Kin #{action == "adding" ? "Added" : "Removed"}: #{fnok_id}" within "#id_of_history_table"}
end

# Usage on libby's user admin page: 
# - Then the history table should show they were added as harrykim's next of kin
# - Then the history table should show they were removed as harrykim's next of kin
Then "the history table should show they were {string} as {string}'s next of kin" do |action, username|
  user_id = User.find_by(login: username).id
  step %{I should see "#{action == "added" ? "Added" : "Removed"} as Fannish Next of Kin for: #{user_id}" within "#id_of_history_table"}
 end
 ```

It looks like we're also missing any sort of test for when the FNOK (in this case, libby) deletes their account. When that happens, the user's (harrykim's) page should say "Removed Fannish Next of Kin: libby's user ID" with a note that it was "System Generated" rather than being done by an admin.